src/app/Fake.BuildServer.AppVeyor/AppVeyorInternal.fs

```diff
@@ -53,10 +53,8 @@ module internal AppVeyorInternal =
                 // because otherwise there might be recursive failure...
                 eprintfn "AppVeyor 'AddMessage' failed: %O" e
 
-    let internal quoteString str =
-        StringBuilder()
-        |> StringBuilder.appendQuotedIfNotNull Some str
-        |> StringBuilder.toText
+    /// quote and escape the single argument, if required.
```

Comment:
all other functions in this file use simple non-xml comments, and it's private anyway